osm2gtfs/creators/fenix/trips_creator_fenix.py

```diff
@@ -54,15 +54,16 @@ def __init__(self, config):
 
         self.exceptions = None
 
-    def add_trips_to_schedule(self, schedule, data):
+    def add_trips_to_feed(self, feed, data):
         routes = data.routes
-        schedule.AddServicePeriodObject(self.service_weekday)
-        schedule.AddServicePeriodObject(self.service_saturday)
-        schedule.AddServicePeriodObject(self.service_sunday)
+        feed.AddServicePeriodObject(self.service_weekday)
+        feed.AddServicePeriodObject(self.service_saturday)
+        feed.AddServicePeriodObject(self.service_sunday)
 
         # Get Fenix data from JSON file
         json_data = []
-        with open('data/linhas.json') as f:
+        schedule_source = self.config.get_schedule_source()
+        with open(schedule_source) as f:
```

Comment:
Did you test this?

Reply:
No, this part I was not able to test. Fenix doesn't run until then, same problem as on master. So, I reverted the changes on this part. Loads the standard file from location.

Reply:
I just applied a fix for Fenix. At least it runs now.

Reply:
It works as before, so this is hopefully not a blocker to merge this in.

Reply:
ok, I finally had the chance to have enough stops cached to get until this point in code execution in Fenix. And I guess we can remove the whole block and replace it with `linhas = data.schedule['data']`. This change is now in the PR.